mindsdb/interfaces/knowledge_base/evaluate.py

```diff
@@ -43,6 +44,19 @@ def calc_entropy(values: List[float]) -> float:
     return -sum([pk * math.log(pk) for pk in values])
 
 
+def sanitize_json_response(response: str) -> str:
+    """Remove markdown code block formatting from JSON response."""
+    # Remove leading/trailing whitespace
+    # Try to find the first JSON object using regex (this assumes a flat or nested dict)
+    json_match = re.search(r"\{.*?\}", response, re.DOTALL)
```

Comment:
for json like `{"a": {"b": 1}}` this regexp will return `{"a": {"b": 1}`
maybe better to look for first `{` and last `}` symbols?